test/index.js

```diff
@@ -634,3 +634,16 @@ test('is.symbol', function (t) {
 
   t.end();
 });
+
+// no need to exhaustively test all negated methods. just need to test structure.
```

Comment:
I added the exhaustive tests, simply by copying the positive tests and negating everything. This did raise the issue that the aliases did not point to the same methods, i.e. `is.not.nil` was a different function from `is.not.null`. I fixed this, even though I don't see the value. At least the cloned tests all pass without having to be removed.